### DISCUSSION TITLE
How do I remove the MHA add-in from my outlook client

### DISCUSSION THREAD

**Discussion by doschecks** [microsoft/MHA#888] [Q&A] [answered]
I'm using Microsoft® Outlook® for Microsoft 365 MSO (Version 2312 Build 16.0.17126.20132) 32-bit. I had MHA install locally on my outlook client, it has been working just fine. However my company asked me to remove it. I couldn't find it this add-in anywhere from my app list or add-in list. I search the web and tried many possible ways to remove the add-in, but I failed. I have no idea how to remove it since I couldn't find it anywhere other than it's visible on my outlook home ribbon. Please help if you know how to remove this add-in. Thanks!

**Comment by Nailer-Git** [comment, marked as answer]
![Remove-MHA](https://github.com/user-attachments/assets/bc20c7d3-712c-4e18-911f-3ed516bed400)

**Reply by braaelahry-wq** [reply]
‏.summay

**Comment by Nailer-Git** [comment]
![2nd-method-to-Remove-MHA](https://github.com/user-attachments/assets/de867744-5b2b-460d-965b-b78199ec2873)
